src/v/cluster/feature_table.cc

```diff
@@ -36,6 +36,8 @@ std::string_view to_string_view(feature f) {
         return "raft_improved_configuration";
     case feature::transaction_ga:
         return "transaction_ga";
+    case feature::raftless_node_status:
+        return "raftless_node_status";
```

Comment:
> the first use case is raft, but generally the new sub-system is not related to raft, right?
True, the feature is not related to the use case. I tried to give a clue in the name that it operates outside of raft, but maybe that's not necessary. Perhaps `node_status` would be enough?

Reply:
i don't have a strong opinion about the naming. i just wanted to point it out in case you had another candidate name that aligned more closely with the feature.